scope.go

```diff
@@ -18,6 +19,80 @@ type resources struct {
 	memory int64
 }
 
+type errStreamOrConnLimitExceeded struct {
```

Comment:
can we put all that stuff at the bottom of the file? 
New readers shouldnt be welcomed with an incomprehensible blob of logging arcana.

Reply:
Moved it to a separate file.